pkg/defaults/policies/files/host_network.json

```diff
@@ -11,6 +11,105 @@
     "DEPLOY"
   ],
   "exclusions": [
+    {
+      "name": "Don't alert on deployment gcp-pd-csi-driver-controller in namespace openshift-cluster-csi-drivers",
+      "deployment": {
+        "name": "gcp-pd-csi-driver-controller",
+        "scope": {
+          "namespace": "openshift-cluster-csi-drivers"
+        }
+      }
+    },
+    {
+      "name": "Don't alert on deployment gcp-pd-csi-driver-node in namespace openshift-cluster-csi-drivers",
+      "deployment": {
+        "name": "gcp-pd-csi-driver-node",
+        "scope": {
+          "namespace": "openshift-cluster-csi-drivers"
+        }
+      }
+    },
+    {
+      "name": "Don't alert on deployment machine-approver in namespace openshift-cluster-machine-approver",
+      "deployment": {
+        "name": "machine-approver",
+        "scope": {
+          "namespace": "openshift-cluster-machine-approver"
```

Comment:
I don't see this exclusion for this policy (Docker CIS 5.9 and 5.20: Ensure that the host's network namespace is not shared)